src/assignment/views/AssignmentEdit.tsx

```diff
@@ -334,7 +315,19 @@ const AssignmentEdit: FunctionComponent<AssignmentEditProps> = ({
 			setLoadingInfo,
 			t('assignment/views/assignment-edit___je-hebt-geen-rechten-om-deze-opdracht-te-bewerken')
 		);
-	}, [loadingInfo, location, match.params, setLoadingInfo, assignmentContent, t, user]);
+	}, [location, match.params, setLoadingInfo, setAssignmentContent, t, user]);
```

Comment:
`React Hook useEffect has missing dependencies: 'currentAssignment' and 'setBothAssignments'.`